web-common/src/features/canvas/components/table/selector.ts

```diff
@@ -46,6 +46,31 @@ export function useComparisonStartEndTime(
   });
 }
 
+export function useStartEndTime(
+  instanceId: string,
+  metricsViewName: string,
+  timeRange: string,
```

Comment:
I'm not following how this `timeRange` is represented. Is this the time range name? Or is it an ISO 8601 time range?

Reply:
This is just a placeholder for now using 8601. Will be replaced by the new time syntax